store/sqlstore/post_store.go

```diff
@@ -2145,6 +2145,17 @@ func (s *SqlPostStore) AnalyticsPostCount(teamId string, mustHaveFile bool, must
 	return v, nil
 }
 
+func (s *SqlPostStore) GetLastPostRow() (lastPost *model.Post, _ error) {
+	query := `SELECT * FROM Posts ORDER BY CREATEAT DESC LIMIT 1`
```

Comment:
I know we discussed ORDER BY DESC on an indexed field as being "ok" - but think it would be good to test this functionality in a db with a few million posts, just to be safe. Wouldn't want this to lock up the DB or anything.

Reply:
It is safe. Also, changed the queries to run even faster